providers/placement/placement.py

```diff
@@ -0,0 +1,159 @@
+        "probability_age_5": Decimal(0.01),
+        "provability_age_16": Decimal(0.01),
+    },
+    "A5": {
+        "probability": Decimal(0.01),
+        "probability_age_5": Decimal(0.01),
+        "provability_age_16": Decimal(0.01),
+    },
+    "A6": {
+        "probability": Decimal(0.01),
+        "probability_age_5": Decimal(0.01),
+        "provability_age_16": Decimal(0.01),
+    },
+    "H5": {
+        "probability": Decimal(0.35),
+        "probability_age_5": Decimal(0),
+        "provability_age_16": Decimal(0),
+    },
+    "K1": {
+        "probability": Decimal(0.005),
+        "probability_age_5": Decimal(0),
+        "provability_age_16": Decimal(0.005),
+    },
+    "K2": {
+        "probability": Decimal(0.005),
+        "probability_age_5": Decimal(0),
+        "provability_age_16": Decimal(0.125),
+    },
+    "P1": {
+        "probability": Decimal(0.005),
+        "probability_age_5": Decimal(0.01),
+        "provability_age_16": Decimal(0.05),
+    },
+    "P2": {
+        "probability": Decimal(0.35),
+        "probability_age_5": Decimal(0),
+        "provability_age_16": Decimal(0),
+    },
+    "P3": {
+        "probability": Decimal(0.01),
+        "probability_age_5": Decimal(0),
+        "provability_age_16": Decimal(0),
+    },
+    "R1": {
+        "probability": Decimal(0.01),
+        "probability_age_5": Decimal(0),
+        "provability_age_16": Decimal(0.01),
+    },
+    "R2": {
+        "probability": Decimal(0.01),
+        "probability_age_5": Decimal(0.01),
+        "provability_age_16": Decimal(0.01),
+    },
+    "R3": {
+        "probability": Decimal(0.0),
+        "probability_age_5": Decimal(0.01),
+        "provability_age_16": Decimal(0),
+    },
+    "R5": {
+        "probability": Decimal(0.005),
+        "probability_age_5": Decimal(0),
+        "provability_age_16": Decimal(0),
+    },
+    "S1": {
+        "probability": Decimal(0.01),
+        "probability_age_5": Decimal(0),
+        "provability_age_16": Decimal(0.01),
+    },
+    "U1": {
+        "probability": Decimal(0.01),
+        "probability_age_5": Decimal(0.01),
+        "provability_age_16": Decimal(0.03),
+    },
+    "U2": {
+        "probability": Decimal(0.01),
+        "probability_age_5": Decimal(0.01),
+        "provability_age_16": Decimal(0.03),
+    },
+    "U3": {
+        "probability": Decimal(0.01),
+        "probability_age_5": Decimal(0.01),
+        "provability_age_16": Decimal(0.03),
+    },
+    "U4": {
+        "probability": Decimal(0.08),
+        "probability_age_5": Decimal(0.445),
+        "provability_age_16": Decimal(0.325),
+    },
+    "U5": {
+        "probability": Decimal(0.08),
+        "probability_age_5": Decimal(0.445),
+        "provability_age_16": Decimal(0.325),
+    },
+    "U6": {
+        "probability": Decimal(0.01),
+        "probability_age_5": Decimal(0.01),
+        "provability_age_16": Decimal(0.01),
+    },
+}
+
+"""    ("A3", 0.01),
```

Comment:
Old list of tuples, I assume this can now just be removed

Reply:
Yep, already removed in the next commit.